runtime/gc_base/modronapi.cpp

```diff
@@ -1134,6 +1134,14 @@ j9gc_reinitialize_for_restore(J9VMThread *vmThread, const char **nlsMsgFormat)
 		goto _error;
 	}
 
+	Assert_MM_true(NULL != extensions->getGlobalCollector());
+
+	if (!extensions->getGlobalCollector()->reinitializeForRestore(env)) {
```

Comment:
in a dilemma if this should be called indirectly though config's reinitialize (and rename its method to be more generic, but no strong opinion

Reply:
Its a bit easier to report the error like this and stay consistent with rest of the reinit.